src/main/java/io/lettuce/core/LMoveArgs.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2011-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.lettuce.core;
+
+import io.lettuce.core.internal.LettuceAssert;
+import io.lettuce.core.protocol.CommandArgs;
+
+import static io.lettuce.core.protocol.CommandKeyword.*;
+
+/**
+ * Argument list builder for the Redis <a href="http://redis.io/commands/blmove">BLMOVE</a> and
+ * <a href="http://redis.io/commands/lmove">LMOVE</a> commands. Static import the methods from {@link Builder} and
+ * chain the method calls: {@code source(LEFT).destination(RIGHT)}.
+ *
+ * @author Mikhael Sokolov
+ */
+public class LMoveArgs implements CompositeArgument {
+
+    public enum Direction {
+        LEFT, RIGHT
+    }
+
+    private Direction source = Direction.RIGHT;
```

Comment:
I am not sure a bit of this defaults for the required args.
@mp911de How do you think we should handle this?

Reply:
Since this is a mandatory argument, let's introduce a constructor that takes the direction argument. That way, `LMoveArgs` isn't required to apply any kind of default.

Reply:
done